### PR TITLE
refactor: migrate unitinput universallink verificationcode and volumesize and delete unselectable

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Refactor

#### What is expected?

Migration of stories to TSX

#### The following changes where made:

(Describe what you did)

1. Migrate unitinput, universallink, verification and volumesize

2. Delete unselectable ⚠️ breaking change

### CODE REVIEW

File: src/components/UnitInput/__stories__/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    storyDescription: `UnitInput provide data as follows:
+
+\`\`\`js
+{
+    unit: [string],
+    value: [string]
+}
+\`\`\``,
+  },
+}
```
Comment (matthprost) [inline src/components/UnitInput/__stories__/index.stories.tsx:33]: Can't you do it this way instead:

```suggestion
Default.parameters = {
  docs: {
    storyDescription: "UnitInput provide data as follows:

```js
{
    unit: [string],
    value: [string]
}
```",
  },
}
```

?

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by matthprost — CHANGES_REQUESTED** [review]
Screenshot 2021-11-16 at 10 03 28

Unit input is a bit broken

File: src/components/UnitInput/__stories__/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+} as Meta
+
+const Template: Story<UnitInputProps> = args => (
+  <UnitInput name="test" {...args} />
+)
+
+export const Default = Template.bind({})
+Default.parameters = {
+  docs: {
+    storyDescription: `UnitInput provide data as follows:
```
Comment (matthprost) [inline src/components/UnitInput/__stories__/index.stories.tsx:24]: Use story.description instead ?

File: src/components/UnitInput/__stories__/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    value: [string]
+}
+\`\`\``,
+  },
+}
+
+export const CustomOptions = Template.bind({})
+CustomOptions.parameters = {
+  docs: {
+    storyDescription: `You can customize options by passing an array of object. Be careful objects should have following structure:
```
Comment (matthprost) [inline src/components/UnitInput/__stories__/index.stories.tsx:38]: Here too

**Review by matthprost — CHANGES_REQUESTED** [review]

**Comment by vincentaudebert** [comment]
> Screenshot 2021-11-16 at 10 03 28
> 
> Unit input is a bit broken

Screenshot 2021-11-16 at 14 55 21

Already the case in production. It is not linked to this MR but I will investigate if it's an easy fix

File: src/components/UnitInput/__stories__/index.stories.tsx (hunk 1 above)
Comment (vincentaudebert) [reply src/components/UnitInput/__stories__/index.stories.tsx:33]: `"` doesn't work with multiline/markdown

**Review by matthprost — APPROVED** [review]

**Comment by DorianMaliszewski** [comment]
Capture d’écran 2021-11-17 à 14 30 50

Can we have props in the first Story because we doens't understand what is supposed to be 😅

**Review by DorianMaliszewski — CHANGES_REQUESTED** [review]
See comment

**Review by DorianMaliszewski — APPROVED** [review]

**Pull request merged**